unit-test/data/tdsTestDomains.json

```diff
@@ -0,0 +1,4 @@
+[
+    {"tracker": "https://google-analytics.com/ga.js", "site": "reddit.com", "req": {"type": "script"}, "result": {"action": "redirect", "firstParty": false, "reason": "matched rule - surrogate"}},
+    {"tracker": "https://google-analytics.com/ga.js", "site": "google.com", "req": {"type": "script"}, "result": {"action": "ignore", "firstParty": true, "reason": "first party"}}
+]
```

Comment:
I'd be cool to add couple more (e.g. facebook) that we know should be blocked (we only cover redirect and ignore above).